browser_bot.py

```
@@ -0,0 +1,604 @@
+                    # Manual mode - wait for user to solve
+                    logger.info("Please solve the CAPTCHA manually...")
+                    self.send_notification("⚠️ CAPTCHA detected! Please solve it manually.")
+
+                    # Wait for CAPTCHA to be solved (check for success indicator)
+                    try:
+                        await self.page.wait_for_selector(
+                            '.recaptcha-success, [data-captcha-solved="true"]',
+                            timeout=120000  # 2 minutes
+                        )
+                        logger.info("CAPTCHA solved!")
+                        return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)